.buildkite/windows.rayci.yaml

```diff
@@ -5,9 +5,10 @@ steps:
     instance_type: builder-windows
 
   - name: ":windows: windows example"
-    job_env: WINDOWS
+    job_env: WIN
```

Comment:
can we not do this? maybe add a version field, or `has_artifacts` field or something? don't hack the `job_env` field.

Reply:
totally, let me see